.github/workflows/build-artifact.yml

```diff
@@ -0,0 +1,69 @@
+name: Build artifact
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  detect-files-changed:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: 'true'
+      - name: Test files changed
+        id: files-changed
+        uses: tj-actions/changed-files@v44
+        with:
+          fetch_additional_submodule_history: 'true'
+          files: |
+            tests/ansibench/*
+            tests/rv8-bench/*
+            tests/*.c
+      - name: Set alias
+        id: has_files_changed
+        run: |
+          if [[ ${{ steps.files-changed.outputs.any_changed }} == true ]]; then
+            echo "has_files_changed=true" >> $GITHUB_OUTPUT
```

Comment:
Description of `any_changed` event, quote from [README](https://github.com/tj-actions/changed-files#output_any_changed):
> Returns `true` when any of the filenames provided using the `files*` or `files_ignore*` inputs have changed. This defaults to `true` when no patterns are specified. i.e. includes a combination of all added, copied, modified and renamed files (ACMR).

Description of `any_modified` event, quote from [README](https://github.com/tj-actions/changed-files#output_any_modified):
>  Returns `true` when any of the filenames provided using the `files*` or `files_ignore*` inputs have been modified. This defaults to `true` when no patterns are specified. i.e. includes a combination of all added, copied, modified, renamed, and deleted files (ACMRD).

I would prefer `any_modified` event instead as it supports deletion. If any test suites are not needed anymore, we can rely on the CI to remove from `sysprog21/rv32emu-prebuilt`.